frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx

```diff
@@ -15,13 +16,16 @@ import Link from "metabase/core/components/Link";
 import ModalContent from "metabase/components/ModalContent";
 import PasswordReveal from "metabase/components/PasswordReveal";
 import { PasswordSuccessMessage } from "./UserSuccessModal.styled";
+import { getSetting } from "metabase/selectors/settings";
 
 class UserSuccessModal extends React.Component {
   componentWillUnmount() {
     this.props.clearTemporaryPassword(this.props.params.userId);
   }
   render() {
-    const { onClose, user, temporaryPassword } = this.props;
+    const { onClose, user, temporaryPassword, isSsoConfigured } = this.props;
+    const isEeSsoConfigured =
+      MetabaseSettings.isEnterprise() && isSsoConfigured;
```

Comment:
I wasn't able to find information in the product doc that the modal copy change should be `EE`-only. Could you double check that or provide a relevant link?

Reply:
From [this resolved comment](https://www.notion.so/Reconsider-how-we-email-new-users-when-SSO-is-active-f3f51ab268ca4c768721fee7d0e0ef03?d=021f4f5aaa8a4dda86d8f03c9369974a#0f6ced4177c546cabdeec73276c8f5ea)

I first make sure how we could provide options for admins to configure this behavior. And it turned out it's to honor the **Enable password authentication** option, which is [only configurable on paid/enterprise plans](https://www.notion.so/metabase/Reconsider-how-we-email-new-users-when-SSO-is-active-f3f51ab268ca4c768721fee7d0e0ef03#3e28045e459043479e1540fb2d4d626e)
![image](https://user-images.githubusercontent.com/1937582/176881132-a22f324d-c147-4b7c-bebe-aa8bfe8c32a9.png)


For OSS, this setting will always return `true`

https://github.com/metabase/metabase/blob/2a502ae3a6f850b0f9dacefed0e1c8d0aad79687/src/metabase/public_settings.clj#L370-L382